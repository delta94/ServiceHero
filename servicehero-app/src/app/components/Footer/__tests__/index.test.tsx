import React from 'react';
import { render } from '@testing-library/react';

import { Footer } from '..';
import { BrowserRouter } from 'react-router-dom';

describe('<Footer  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
