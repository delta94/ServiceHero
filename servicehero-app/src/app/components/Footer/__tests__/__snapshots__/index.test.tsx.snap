// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer  /> should match snapshot 1`] = `
.c2 {
  margin-right: 40px;
}

.c1 {
  display: inline-block;
  margin: 0px 5px;
  color: grey;
}

.c0 {
  padding: 0px;
  padding-left: 60px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: left;
  -webkit-justify-content: left;
  -ms-flex-pack: left;
  justify-content: left;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
  height: 40px;
  color: grey;
}

<footer
  class="ant-layout-footer c0"
>
  <a
    class="c1"
    href="/"
  >
    ServiceHero 
  </a>
  <span
    class="c2"
  >
    Â© 2021
  </span>
</footer>
`;
